server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DataTransferRuntimeFlowableTask.java

```diff
@@ -86,7 +86,7 @@ protected Void start(Long taskId, TaskService taskService, DelegateExecution exe
             odcInternalFileService.getExternalImportFiles(taskEntity, submitter, config.getImportFileName());
         }
         context = dataTransferService.create(taskId + "", config);
-        taskService.start(taskId, DataTransferTaskResult.of(context));
```

Comment:
why don't we initialize the result of it here, but add a hook at flowinstanceService

Reply:
Because when the program reaches the start method of this class, it indicates that the approval has been passed and it has entered the execution phase. But our goal is to show the results to users during approval.
So we can only add a hook to initialize the results when creating the task.